### ISSUE TITLE
Add option to flag outdated packages

### ISSUE THREAD

**Issue by baimafeima** [solus-project/solus-sc#103]
Is there a way to implement an option to flag outdated packages directly via solus-sc?

**Comment by ikeydoherty** [comment]
Eh. Smells too much like Arch Linux to me..

**Comment by baimafeima** [comment]
Yes. I think it would make sense if there is a way to prioritize at what speed software gets updated and if there is a way to automatically generate an issue on Phabricator. The higher the flag count, the higher the priority. Not sure if this would invite misuse though.

**Comment by ikeydoherty** [comment]
Oh it'd invite misuse. I think the existing mechanism is more than sufficient, no? Opening an issue directly on phab - and it deliberately has some blockers in the way, in that you need to verify an account first before opening an issue, so it puts off the anonymous stuff.

**Comment by baimafeima** [comment]
> Opening an issue directly on phab - and it deliberately has some blockers in the way, in that you need to verify an account first before opening an issue, so it puts off the anonymous stuff.

Yeah, you're right. The new link in the _License tab_ "Report an invalid or missing license" is great by the way. I think this is extremely thoughtful and could prevent legal issues before they even have a chance to be raised. How about a similar link for outdated packages: "Report broken or outdated package"?

**Comment by ikeydoherty** [comment]
Where is the best place to put that, though?

**Comment by tristan957** [comment]
It could be in the details pane in a similar way that the License flag is.
